README.ja.md

```
@@ -365,6 +365,8 @@ const list = ref([
 
 ### `useSpring` composable
 
+> **非推奨**。代わりに [`<spring>` コンポーネント](#spring-component) を使用してください。`realValue` / `realVelocity` が必要な場合は [`springValue`](#springvalueinitial) を使用してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)